core/vm/stack.go

```diff
@@ -18,36 +18,36 @@ package vm
 
 import (
 	"fmt"
-	"math/big"
+
+	"github.com/holiman/uint256"
 )
 
 // Stack is an object for basic stack operations. Items popped to the stack are
 // expected to be changed and modified. stack does not take care of adding newly
 // initialised objects.
 type Stack struct {
-	data []*big.Int
+	data []uint256.Int
 }
 
 func newstack() *Stack {
-	return &Stack{data: make([]*big.Int, 0, 1024)}
+	return &Stack{data: make([]uint256.Int, 0, 16)}
```

Comment:
1024 -> 16? is this right?

Reply:
It seems so, yes. Remember that number is the capacity of the slice, that means the preallocated size, not necessarily the max or current size

Reply:
https://github.com/ethereum/go-ethereum/blob/master/core/vm/stack.go#L28